problems.txt

```diff
@@ -0,0 +1,96 @@
+2017-07-06T17:31:58.698Z pid:20901 worker:0 INFO supervisor started worker 1 (pid 20915)
+2017-07-06T17:31:58.698Z pid:20901 worker:0 INFO supervisor resized to 1
+2017-07-06T17:31:58.910Z pid:20915 worker:1 [Thu Jul  6 10:31:58 2017] com.ibm.diagnostics.healthcenter.loader INFO: Node Application Metrics 3.0.2.201706071954 (Agent Core 3.2.1)
+2017-07-06T17:31:58.959Z pid:20915 worker:1 [Thu Jul  6 10:31:58 2017] com.ibm.diagnostics.healthcenter.mqtt INFO: Connecting to broker localhost:1883
+2017-07-06T17:31:58.972Z pid:20915 worker:1 Starting ibmapm for gateway "https://localhost:443"
+2017-07-06T17:31:58.990Z pid:20915 worker:1 [2017-07-06 10:31:58.989] [INFO] knj_log - KNJ_LOG_LEVEL is not set or not set correctly through environment variables.
+2017-07-06T17:31:58.991Z pid:20915 worker:1 [2017-07-06 10:31:58.990] [INFO] knj_log - The program set default log level to INFO.
+2017-07-06T17:31:58.998Z pid:20915 worker:1 [2017-07-06 10:31:58.997] [INFO] knj_log - ==========Inital parameters setting==========
+2017-07-06T17:31:58.998Z pid:20915 worker:1 [2017-07-06 10:31:58.998] [INFO] knj_log - Monitoring server type:  BAM
+2017-07-06T17:31:58.999Z pid:20915 worker:1 [2017-07-06 10:31:58.998] [INFO] knj_log - Monitoring server url:  undefined
+2017-07-06T17:31:59.000Z pid:20915 worker:1 [2017-07-06 10:31:58.998] [INFO] knj_log - Monitoring application name: undefined
+2017-07-06T17:31:59.000Z pid:20915 worker:1 [2017-07-06 10:31:58.998] [INFO] knj_log - Monitoring security url: undefined
+2017-07-06T17:31:59.000Z pid:20915 worker:1 [2017-07-06 10:31:58.998] [INFO] knj_log - Monitoring server SNI(Server Name Indication): undefined
+2017-07-06T17:31:59.000Z pid:20915 worker:1 [2017-07-06 10:31:58.999] [INFO] knj_log - ==========End of inital parameters setting==========
+2017-07-06T17:31:59.000Z pid:20915 worker:1 [2017-07-06 10:31:58.999] [INFO] knj_log - ==========Inital BAM parameters setting==========
+2017-07-06T17:31:59.001Z pid:20915 worker:1 [2017-07-06 10:31:58.999] [INFO] knj_log - KNJ_AAR_BATCH_COUNT undefined
+2017-07-06T17:31:59.001Z pid:20915 worker:1 [2017-07-06 10:31:59.000] [INFO] knj_log - KNJ_AAR_BATCH_FREQ undefined
+2017-07-06T17:31:59.001Z pid:20915 worker:1 [2017-07-06 10:31:59.000] [INFO] knj_log - KNJ_AAR_BATCH_COUNT 100
+2017-07-06T17:31:59.001Z pid:20915 worker:1 [2017-07-06 10:31:59.000] [INFO] knj_log - KNJ_AAR_BATCH_FREQ 60
+2017-07-06T17:31:59.001Z pid:20915 worker:1 [2017-07-06 10:31:59.000] [INFO] knj_log - ==========End of inital BAM parameters setting==========
+2017-07-06T17:31:59.166Z pid:20915 worker:1 [2017-07-06 10:31:59.165] [INFO] knj_log - Configuration updated: deepdive is enabled
+2017-07-06T17:31:59.176Z pid:20915 worker:1 [2017-07-06 10:31:59.168] [ERROR] knj_restclient_log - register_topology set cofniguration failed. { Error: ENOENT: no such file or directory, open './config.json'
+2017-07-06T17:31:59.176Z pid:20915 worker:1     at Object.fs.openSync (fs.js:651:18)
```

Comment:
This stack trace shouldn't be here, only the `err.message` should be printed to stdout (user's don't care about the stack).